src/main/java/com/WWI16AMA/backend_api/Plane/Plane.java

```diff
@@ -30,6 +31,10 @@
     @PositiveOrZero
     private double pricePerFlightMinute;
 
+    @OneToOne(cascade = CascadeType.ALL)
+    //@JsonIgnoreProperties({"entrys"})
+    private PlaneLog planeLog;
```

Comment:
hier stattdessen irgendwas in die Richtung:
```
@OneToMany(CascadeType.ALL, FetchTyp.LAZY)
@JsonIgnore
List<PlaneLogEntry> planeLog;
```
_achtung: snippet sinngemäß_

Reply:
Json ignored das jetzt und der PlaneLog ist als Liste im Plane direkt gespeichert.